atlasdb-config/src/main/java/com/palantir/atlasdb/http/AtlasDbHttpClients.java

```diff
@@ -80,6 +83,14 @@ private AtlasDbHttpClients() {
         return wrapWithOkHttpBugHandling(type, clientFactory);
     }
 
+    public static <T> T createDialogueProxy(TaggedMetricRegistry registry, Class<T> type, Channel channel) {
```

Comment:
I'd recommend putting the word 'shim' in here so that it's super clear which ones are using the 'full dialogue' codegen vs the feign thingy.

Reply:
Makes sense